src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 0) {
+    if (days < 3) {
+      return Number(days) * 40;
+    }
+
+    if (days < 7) {
+      return Number(days) * 40 - 20;
+    }
+
+    return days * 40 - 50;
```

Comment:
please try to add variables when using some specific numbers, when someone will do your code review, they also need to know what those numbers mean. In our case it's obvious, but typically it's a good practice